src/azure-cli/azure/cli/command_modules/servicefabric/_help.py

```diff
@@ -31,6 +60,162 @@
         az sf application certificate add -g group-name -n cluster1  --secret-identifier 'https://{KeyVault}.vault.azure.net/secrets/{Secret}'
 """
 
+helps['sf application show'] = """
+type: command
+short-summary: Get an application.
+examples:
+  - name: Get application.
```

Comment:
Too short. Consider "Show the properties of an application in Azure Service Fabric cluster".